src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx

```diff
@@ -111,7 +111,11 @@ export const ShowFeelingTemplate: React.FC<ShowFeelingTemplateProps> = ({
               >
                 <div className="feelingOfDay-name">
                   {feelingsEmojis[feelingsListObject[feelingId].category]}&nbsp;
-                  <span>{t(feelingsListObject[feelingId].content)}</span>
+                  <div>
+                    <span>{t(feelingsListObject[feelingId].content)}</span>
+                    <br />
+                    <span style={{ fontSize: "1rem", opacity: "0.8" }}>{t(feelingsListObject[feelingId].note)}</span>
```

Comment:
and change this css to -> ` {
    font-size: 0.875rem;
    opacity: 0.8;
    word-break: break-word;
 } `

Reply:
Got it